lib/Common/Common/vtinfo.h

```diff
@@ -87,22 +87,24 @@ class VirtualTableInfoBase
 {
 public:
     static INT_PTR GetVirtualTable(void * ptr) { return (*(INT_PTR*)ptr); }
+protected:
+    static void SetVirtualTable(void * ptr, INT_PTR vt) { *(INT_PTR*)ptr = vt; }
```

Comment:
> Why changing the replacement new to this direct assignment?

Described in commit message. "placement new" has undesired side effects -- it calls field default constructors.